### PR TITLE
My Jetpack: minor doc improvement

### PR DESCRIPTION
**Pull request by retrofox**
Add testing development instructions

Fixes #

#### Changes proposed in this Pull Request:

* My Jetpack: minor doc improvement

#### Does this pull request change what data or activity we track or use?

no

#### Testing instructions:

* Go to readme file. Check the changes :smart:

### CODE REVIEW

File: projects/packages/my-jetpack/README.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ WP Admin page with information and configuration shared among all Jetpack stand-
 
 Every Jetpack plugin must include the My Jetpack package.
 
+Define the `JETPACK_ENABLE_MY_JETPACK` constante as True:
```
Comment (jeherve) [inline projects/packages/my-jetpack/README.md:9]:
```suggestion
Define the `JETPACK_ENABLE_MY_JETPACK` constant as true:
```

**Review by jeherve — CHANGES_REQUESTED** [review]
You'll need to update versions I"m afraid, with `tools/fixup-project-versions.sh`

**Comment by retrofox** [comment]
> You'll need to update versions I"m afraid, with `tools/fixup-project-versions.sh`

sorry, why? 😅

**Comment by jeherve** [comment]
> > You'll need to update versions I"m afraid, with `tools/fixup-project-versions.sh`
> 
> sorry, why? 😅

That's because you marked your change as "minor", not "patch", so that forces a version bump.

**Comment by retrofox** [comment]
> > > You'll need to update versions I"m afraid, with `tools/fixup-project-versions.sh`
> > 
> > 
> > sorry, why? 😅
> 
> That's because you marked your change as "minor", not "patch", so that forces a version bump.

What's the right level for this change?

**Comment by jeherve** [comment]
Personally I would have marked it as "patch", but that's up to you!

**Review by jeherve — APPROVED** [review]

**Pull request merged**